manifests/pipecd/templates/deployment.yaml

```diff
@@ -252,36 +252,42 @@ spec:
 
 {{- if .Values.quickstart}}
 ---
+# MySQL datastore
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "pipecd.fullname" . }}-mongodb
+  name: {{ include "pipecd.fullname" . }}-mysql
   labels:
     {{- include "pipecd.labels" . | nindent 4 }}
-    app.kubernetes.io/component: mongodb
+    app.kubernetes.io/component: mysql
 spec:
   replicas: 1
   selector:
     matchLabels:
       {{- include "pipecd.selectorLabels" . | nindent 6 }}
-      app.kubernetes.io/component: mongodb
+      app.kubernetes.io/component: mysql
   template:
     metadata:
       labels:
         {{- include "pipecd.selectorLabels" . | nindent 8 }}
-        app.kubernetes.io/component: mongodb
+        app.kubernetes.io/component: mysql
     spec:
       containers:
-        - name: mongodb
-          image: mongo:{{ .Values.mongodb.imageTag }}
+        - name: mysql
+          image: mysql:{{ .Values.mysql.imageTag }}
           imagePullPolicy: IfNotPresent
+          env:
+            - name: MYSQL_ROOT_PASSWORD
+              value: test
+            - name: MYSQL_DATABASE
+              value: quickstart
```

Comment:
Those values should be specified in the `values.yaml` file. (Same with `imageTag`)

Reply:
Addressed on https://github.com/pipe-cd/pipe/pull/1742/commits/8ad14197a34cdc761165ea9b0fe0fe2d18263240 🙏